packages/cactus-cmd-api-server/src/test/typescript/unit/get-open-api-spec-v1-oauth2-scopes.test.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  ApiServer,
+  ApiServerApiClient,
+  ApiServerApiClientConfiguration,
+  AuthorizationProtocol,
+  ConfigService,
+  IAuthorizationConfig,
+} from "../../../main/typescript/public-api";
+import {
+  IJoseFittingJwtParams,
+  LogLevelDesc,
+} from "@hyperledger/cactus-common";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+import { Constants } from "@hyperledger/cactus-core-api";
+import type { AuthorizeOptions as SocketIoJwtOptions } from "@thream/socketio-jwt";
+import type { Params as ExpressJwtOptions } from "express-jwt";
+import "jest-extended";
+import { SignJWT, exportSPKI, generateKeyPair } from "jose";
+import path from "path";
+import { v4 as uuidv4 } from "uuid";
+
+interface IExpectedScopes {
+  [key: string]: string;
+}
+
+describe("cmd-api-server:getOpenApiSpecV1Endpoint", () => {
+  const logLevel: LogLevelDesc = "TRACE";
```

Comment:
Change log level to `info`, dont use `trace` for tests in CI

Reply:
@outSH Got it, just finished updating it.